sdk/src/main/java/io/radar/sdk/model/RadarLog.kt

```diff
@@ -22,9 +22,11 @@ internal data class RadarLog(
 
         @JvmStatic
         fun fromJson(json: JSONObject): RadarLog {
+            val levelString = json.optString(LEVEL)
+            val typeString = json.optString(TYPE)
             return RadarLog(
-                level = Radar.RadarLogLevel.valueOf(json.optString(LEVEL)),
-                type = Radar.RadarLogType.valueOf(json.optString(TYPE)),
+                level = if (levelString.isNotBlank()) Radar.RadarLogLevel.valueOf(levelString) else Radar.RadarLogLevel.INFO,
+                type = if (typeString.isNotBlank() && typeString!="NONE") Radar.RadarLogType.valueOf(typeString) else null,
```

Comment:
Is the old code causing issues? ie, does it not handle blanks or NONE and now we're generating those values?

Reply:
Yes the old code was causing issues as it did not handle the enum NONE properly.